src/data/builder-tools.js

```diff
@@ -1091,6 +1091,14 @@ export const Showcases = [
     getstarted:"https://mazzumagpt.gitbook.io/mazzumagpt-docs/",
     tags: ["plutus"]
   },
+  {
+    title: "CardanoPress",
+    description: "Bridging Web2 and Web3 by combining WordPress and Cardano",
+    preview: require("./builder-tools/cardanopress-dev-portal-banner-355x148.png"),
+    website: "https://cardanopress.io",
+    getstarted:"https://cardanopress.io/docs/",
+    tags: ["hosted", "http", "nft"]
```

Comment:
Does it have published API for using `http` tag?

Reply:
HTTP tag in the builder tool category is for APIs that expose access over HTTP. But I think the problem here is that the tags can not fit because the category does not fit. (see comment below with recommendation to move into showcase section)